docs/source/index.md

```diff
@@ -1,9 +1,41 @@
-# Jupyter Project Documentation
+# Project Jupyter Documentation
 
-**Welcome to the Jupyter Project documentation**. This website acts as "meta" documentation for the Jupyter ecosystem.
-It has a collection of resources to navigate the tools and communities in this ecosystem, and to help you get started.
+Welcome to the Project Jupyter documentation site. Jupyter is a large umbrella
+project that covers many different software offerings and tools, including the
+popular [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/)
+and [JuyterLab](https://jupyterlab.readthedocs.io/en/stable/) web-based
+notebook authoring and editing applications. The Jupyter project and its
+subprojects all center around providing tools for interactive computing with
+computational notebooks.
 
-## Start Here
+## What is a Notebook?
+
+![jupyterlab.png](_static/_images/jupyterlab.png)
+
+**Pictured:** *A computational notebook document, shown inside JupyterLab*
+
+A notebook is a shareable document that combines computer code, plain language
+descriptions, data, rich visualizations like 3D models, charts, graphs and
+figures, and interactive controls. A notebook, along with an editor (like
+JupyterLab), provides a fast interactive environment for prototyping and
+explaining code, exploring and visualizing data, and sharing ideas with
+others.
+
+## Where do I start?
+
+Most people begin with Jupyter by installing an editing application that fits
+their preferences, like [JuyterLab](https://jupyterlab.readthedocs.io/en/stable/)
+or [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/),
+and making their first notebook document:
+
+- Jupyter Notebook offers a simplified, lightweight notebook authoring experience
+- JupyterLab offers a feature-rich, tabbed multi-notebook editing environment
```

Comment:
Probably worth ending the list with "and other interfaces see https://docs.jupyter.org/en/latest/projects/user-interfaces.html" (and updating that list)

Reply:
Added this.